defines.mk

```diff
@@ -2,9 +2,6 @@
 # using the -D flag. Many of the #defines are based on the current CPU
 # configuration found in config.mk. 
 
-
-
-
 include ${FLEXPRET_ROOT_DIR}/config.mk
 
 PRINTF_DEFINES := -D PRINTF_ALIAS_STANDARD_FUNCTION_NAMES_SOFT \
```

Comment:
Considering the size of printf. It would be good with information somewhere on how to disable it. Is that information brief could it maybe fit here?

Reply:
There wasn't really an easy way to do that but I fixed it in the last commit :)